core/trino-main/src/test/java/io/trino/operator/scalar/TestMathFunctions.java

```diff
@@ -3457,6 +3457,25 @@ public void testCosineSimilarity()
                 .isNull(DOUBLE);
     }
 
+    @Test
+    public void testCosineDistance()
```

Comment:
Not sure if we really need those tests ... in separate method or at all. Will see how they run and decide later